{\rtf1\ansi\ansicpg1252\cocoartf2759
\cocoatextscaling0\cocoaplatform0{\fonttbl\f0\froman\fcharset0 Times-Bold;\f1\froman\fcharset0 Times-Roman;}
{\colortbl;\red255\green255\blue255;\red0\green0\blue0;}
{\*\expandedcolortbl;;\cssrgb\c0\c0\c0;}
\margl1440\margr1440\vieww11520\viewh8400\viewkind0
\deftab720
\pard\pardeftab720\sa321\partightenfactor0

\f0\b\fs48 \cf0 \expnd0\expndtw0\kerning0
\outl0\strokewidth0 \strokec2 scRNA-Seq Practice\
\pard\pardeftab720\sa298\partightenfactor0

\fs36 \cf0 Introduction\
\pard\pardeftab720\sa240\partightenfactor0

\f1\b0\fs24 \cf0 Welcome to my scRNA-Seq Practice repository! I am a fifth-year PhD student at Rutgers University, Waksman Institute of Microbiology. My passion for bioinformatics began with my PhD thesis, which focuses on optimizing and analyzing translation initiation profiling and RNA-seq data. Recently, I've expanded my expertise into single-cell RNA sequencing (scRNA-seq) techniques, fueled by a workshop I attended at MSKCC and my broader academic interests.\
\pard\pardeftab720\sa298\partightenfactor0

\f0\b\fs36 \cf0 Project Description\
\pard\pardeftab720\sa240\partightenfactor0

\f1\b0\fs24 \cf0 This repository is a collection of my efforts to deepen my understanding and skills in scRNA-seq. Inspired by cutting-edge training and my commitment to pursuing a career in this dynamic field, I recreate analyses from published papers here. Each project within this repository represents a comprehensive exercise in replicating and understanding the intricate data manipulations and visualizations that drive contemporary bioinformatics research in scRNA-seq.\
\pard\pardeftab720\sa298\partightenfactor0

\f0\b\fs36 \cf0 Acknowledgments\
\pard\pardeftab720\sa240\partightenfactor0

\f1\b0\fs24 \cf0 Special thanks to the experts and organizers at MSKCC for providing a robust learning experience that has significantly influenced this work.\
}